### PR TITLE
Fixed proxy merge conflicts

### PR DESCRIPTION
**Pull request by DavidTHoy**

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -3,11 +3,18 @@
 from webex_bot.commands.echo import EchoCommand
 from webex_bot.webex_bot import WebexBot
 
+# (Optional) Proxy configuration
+proxies = {
+    'https': 'http://proxy.esl.example.com:80',
```
Comment (fbradyirl) [inline example.py:8]: Does https need to be set? I dont see it being used.

**Comment by fbradyirl** [comment]
Also could you add a note to the README to indicate how to set this up?

File: example.py (hunk 1 above)
Comment (DavidTHoy) [reply example.py:8]: This is just an example. Per the WebexPythonSDK documentation at [here](https://github.com/WebexCommunity/WebexPythonSDK/blob/0f1305b2580e38ec6e04972750a4791d9f77f15c/docs/user/quickstart.rst#L160)

It should be set like so:
`from webexteamssdk import WebexTeamsAPI
proxy = {'https': 'http://<proxy_ip>:<proxy_port>'}
api = WebexTeamsAPI(access_token=<your_access_token>, proxies=proxy)`

Comment (fbradyirl) [reply example.py:8]: I think if `wss` is the only one that will be used, then only use that in the example and also show in README. Some folks might be confused if they set a https  proxy and then things dont work.

Comment (fbradyirl) [reply example.py:8]: Ok I see `WebexTeamsAPI` is also using this, so both are actually required for proxy support.

**Review by fbradyirl — APPROVED** [review]

**Pull request merged**